internal/pkg/ccloudv2/flink.go

```diff
@@ -59,3 +86,34 @@ func (c *Client) UpdateFlinkComputePool(id string, update flinkv2.FcpmV2ComputeP
 	res, httpResp, err := c.FlinkClient.ComputePoolsFcpmV2Api.UpdateFcpmV2ComputePool(c.flinkApiContext(), id).FcpmV2ComputePoolUpdate(update).Execute()
 	return res, errors.CatchCCloudV2Error(err, httpResp)
 }
+
+func (c *Client) CreateFlinkIAMBinding(region, cloud, environmentId, identityPoolId string) (flinkv2.FcpmV2IamBinding, error) {
+	iamBinding := flinkv2.FcpmV2IamBinding{
+		Region:       flinkv2.PtrString(region),
+		Cloud:        flinkv2.PtrString(cloud),
+		Environment:  flinkv2.NewGlobalObjectReference(environmentId, "", ""),
+		IdentityPool: flinkv2.NewGlobalObjectReference(identityPoolId, "", ""),
+	}
+	res, httpResp, err := c.FlinkClient.IamBindingsFcpmV2Api.CreateFcpmV2IamBinding(c.flinkApiContext()).FcpmV2IamBinding(iamBinding).Execute()
+	return res, errors.CatchCCloudV2Error(err, httpResp)
+}
+
+func (c *Client) DeleteFlinkIAMBinding(id, environmentId string) error {
+	httpResp, err := c.FlinkClient.IamBindingsFcpmV2Api.DeleteFcpmV2IamBinding(c.flinkApiContext(), id).Environment(environmentId).Execute()
+	return errors.CatchCCloudV2Error(err, httpResp)
+}
+
+func (c *Client) ListFlinkIAMBindings(environmentId string, params *ListFlinkIAMBindingsQueryParams) ([]flinkv2.FcpmV2IamBinding, error) {
```

Comment:
Can you pass the individual fields instead of the param struct? We want to auto-generate this code in the future, and this is how it will be styled

Reply:
Changed!